build-tools/create-packs/Microsoft.Android.Sdk.proj

```diff
@@ -69,6 +69,9 @@ core workload SDK packs imported by WorkloadManifest.targets.
       <_PackageFiles Include="$(XAInstallPrefix)xbuild\Xamarin\Android\Microsoft.Android.Sdk.ILLink.pdb" PackagePath="tools" />
       <_PackageFiles Include="$(XAInstallPrefix)xbuild\Xamarin\Android\%(_LocalizationLanguages.Identity)\Microsoft.Android.Sdk.ILLink.resources.dll" PackagePath="tools\%(_LocalizationLanguages.Identity)" />
       <_PackageFiles Include="$(ToolsSourceDir)**" PackagePath="tools" />
+      <_PackageFiles Include="$(NetCoreAppToolsSourceDir)class-parse.dll" PackagePath="tools" />
+      <_PackageFiles Include="$(NetCoreAppToolsSourceDir)class-parse.pdb" PackagePath="tools" />
+      <_PackageFiles Include="$(NetCoreAppToolsSourceDir)class-parse.runtimeconfig.json" PackagePath="tools" />
```

Comment:
For classic Xamarin.Android, is there also a `class-parse.exe` you need here:

https://github.com/xamarin/xamarin-android/blob/f98c4a8ad4b8703cf9bb39b383b44b1759b49acd/build-tools/installers/create-installers.targets#L130-L131

Reply:
I would have thought so, but we were already shipping `class-parse.exe` in Classic even though we weren't using it:

https://github.com/xamarin/xamarin-android/blob/f98c4a8ad4b8703cf9bb39b383b44b1759b49acd/build-tools/installers/create-installers.targets#L128-L129